### ISSUE TITLE
Module path should be "github.com/ConsenSys/gnark", not "github.com/consensys/gnark"

### ISSUE THREAD

**Issue by KateGo520** [ConsenSys/gnark#27]
## Background
Module path is inconsistent with go import path.
GO111MODULE=on, run `go get github.com/ConsenSys/gnark`:

```
go: downloading github.com/ConsenSys/gnark v0.2.1-alpha
go: github.com/ConsenSys/gnark upgrade => v0.2.1-alpha
go get: github.com/ConsenSys/gnark@v0.2.1-alpha: parsing go.mod:
        module declares its path as: github.com/consensys/gnark
                but was required as: github.com/ConsenSys/gnark
```

###  Fix the module path:
1) Consider reverting the `consensys` -> `ConsenSys` rename.
2) Rename the module path to ”github.com/ConsenSys/gnark”: https://github.com/ConsenSys/gnark/blob/master/go.mod#L1 
```
module github.com/ConsenSys/gnark
go 1.14
require (
	…
) 
```
3) Example `go get github.com/consensys/gnark` commands or others in the READMEs of all the go repos to guide consumers.

**Comment by KateGo520** [comment]
@bot @ThomasPiellard  Could you help me review this issue? Thx :p

**Comment by gbotrel** [comment]
@KateGo520  thanks for flagging that, all this will be homogenized in our next release 👍

**Comment by gbotrel** [comment]
Fixed in v0.3.X (now merged in `master`) - please use lower case module path name (`github.com/consensys/gnark` )
